conf/app.routes

```diff
@@ -9,24 +9,14 @@ GET           /admin/metrics                    com.kenshoo.play.metrics.Metrics
 
 # Note that these resources take up to four query parameters: postcode, filter, fuzzy, uprn
 # and the tests are more explicit as a result.
-GET           /gb/addresses                     osgb.AddressSearchController.searchV1()
-GET           /uk/addresses                     osgb.AddressSearchController.searchV1()
-GET           /v1/gb/addresses                  osgb.AddressSearchController.searchV1()
-GET           /v1/uk/addresses                  osgb.AddressSearchController.searchV1()
-GET           /v2/gb/addresses                  osgb.AddressSearchController.searchV2()
-GET           /v2/uk/addresses                  osgb.AddressSearchController.searchV2()
+GET           /v2/gb/addresses                  osgb.AddressSearchController.search()
+GET           /v2/uk/addresses                  osgb.AddressSearchController.search()
```

Comment:
Do we need both? Does anybody use gb?

As far as I';m aware there is now no difference, we don't exclude NI data from the GB endpoint